client/src/utils/deprecation.ts

```diff
@@ -0,0 +1,50 @@
+const NEXT_VERSION = '5.0';
```

Comment:
Question: I notice this duplicates where the version of Wagtail is defined. Is there a way this could be populated from the backend from the package itself?